Gemfile

```diff
@@ -22,7 +22,7 @@ gem "govuk_frontend_toolkit", "0.44.0"
 if ENV["GOVSPEAK_DEV"]
   gem "govspeak", path: "../govspeak"
 else
-  gem "govspeak", "3.4.0"
+  gem "govspeak", "~> 3.5"
```

Comment:
Did you mean to allow `minor` version updates, rather than `patch` (e.g. `~> 3.5.0`) ?

Technically, this should be fine if we follow semver properly. Just want to check.